gatsby-image-gallery/src/row.tsx

```diff
@@ -1,9 +1,17 @@
-import styled from 'styled-components'
+import styled, { css } from 'styled-components'
 
-const Row = styled.div`
+interface RowProps {
+  margin: number
+}
+
+const Row = styled.div<RowProps>`
   display: flex;
   flex-wrap: wrap;
-  margin-right: -15px;
-  margin-left: -15px;
+  ${(props) => {
+    return css`
+      margin-right: ${props.margin || -15}px;
+      margin-left: ${props.margin || -15}px;
```

Comment:
This doesn't let one set the margin to 0, does it? Might be better to set it as optional and check for `undefined` before returning this...